assets/src/blocks/CarouselHeader/EditableBackground.js

```diff
@@ -0,0 +1,53 @@
+import {MediaUpload} from "@wordpress/block-editor";
+import {ImagePlaceholder} from './ImagePlaceholder';
+import {EditableBackgroundActions} from './EditableBackgroundActions';
+
+export const EditableBackground = ({
+  image_url,
+  image_id,
+  index,
+  slides,
+  setAttributes,
+  focalPoints,
+  children,
+}) => {
+  function onImageChange(index, value) {
+    let slidesCopy = JSON.parse(JSON.stringify(slides));
+    if (null !== value) {
+      slidesCopy[index].image = value;
+    } else {
+      slidesCopy[index].image = null;
```

Comment:
Set it to the value, unless the value is `null`, then set it to `null` instead of `null` :thinking: